languages/csharp/reference/exercise-concepts/leap.md

```diff
@@ -0,0 +1,19 @@
+# Concepts of Leap
+
+[Example implementation](https://github.com/exercism/csharp/blob/master/exercises/leap/Example.cs)
+
+## General
```

Comment:
Two concepts I'm missing are type-related:

- integers
- booleans

The example implementation also uses the `%` operator, which is a math operator.